styles/Vocab/Stakater/accept.txt

```diff
@@ -4,6 +4,8 @@
 [Aa]utoscale
 [Aa]utoscalers
 [Aa]utoscaling
+[Bb]aremetal
+bare-metal
```

Comment:
why not just standardize on baremetal?

Reply:
I've seen  both in lots of places, but it can leave just one, as standardization for Stakater's documents